api/src/main/java/io/opentelemetry/trace/TraceVersion.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.trace;
+
+public class TraceVersion {
```

Comment:
I'm not sure we need a whole class just for one method. Could this just be a method on the HttpTraceContext itself?

Reply:
because `BigendianEncoding` class is package private and I cannot call it.

Reply:
It's just two bytes - I think I would just generate a `Set` of all possible versions as a private in `HttpTraceContext` (255 elements only) and use that for validation, seems like it could be faster anyways.

Reply:
done